spring-batch-docs/asciidoc/readersAndWriters.adoc

```diff
@@ -1460,7 +1462,7 @@ The 'trade' tag is defined as the 'root element' in the scenario above. Everythi
 between '&lt;trade&gt;' and '&lt;/trade&gt;' is considered one 'fragment'. Spring Batch
 uses Object/XML Mapping (OXM) to bind fragments to objects. However, Spring Batch is not
 tied to any particular XML binding technology. Typical use is to delegate to
-link:$$http://docs.spring.io/spring-ws/site/reference/html/oxm.html$$[Spring OXM], which
+link:$$https://docs.spring.io/spring/docs/current/spring-framework-reference/data-access.html#oxm$$[Spring OXM], which
```

Comment:
Again, we should do this with a variable. Take a look at how Spring Boot does it.

Reply:
While I completely agree on the previous remark regarding the hard coded link to the master branch for the samples module (for which I will update the link to use an asciidoctor variable with the current version), this link to Spring OXM can still point to the current version of SF. We use the same pattern in other places like the section about "FlatFileItemWriter" where we have: 

```
Documentation explaining how to create beans of this type can be found in
link:$$https://docs.spring.io/spring/docs/current/spring-framework-reference/core.html#resources$$[Spring
Framework, Chapter 5. Resources]
```

So I guess since this is not a link to a Spring Batch related item, it can remain like this for now and update all the links to use variables in another pass. That said, I will still update the current PR to use a variable in the link to the samples module

Reply:
Same as previous comment, changing all the links to use variables can be done in another PR. This small change just updates the link from the old 'spring-ws' site to the new one.